src/Sylius/Bundle/AddressingBundle/EventListener/ZoneMemberIntegrityListener.php

```diff
@@ -71,7 +71,9 @@ private function getSession(): SessionInterface
     {
         // bc-layer for Symfony 4
         if (!method_exists(RequestStack::class, 'getSession')) {
-            return $this->requestStack->getMasterRequest()->getSession();
+            $request = $this->requestStack->getMasterRequest();
+            Assert::notNull($request);
+            return $request->getSession();
```

Comment:
Missing empty space before `return`

Reply:
Is this kind of mistake not covered by the cron task handling ECS ?

Reply:
I'm wondering myself each time I'm seeing it, but ECS check is not perfect right now.